client/src/pages/Map/Map.js

```diff
@@ -89,16 +89,28 @@ export default function Map({
   const [isMapLoaded, setIsMapLoaded] = useState(false);
   const selectionEnabledRef = useRef(selectionEnabled);
   const currentFeature = useRef(null);
-  const hasInitialFlyToFired = useRef(false);
 
   const initialLoadTreeId = useRef(
     new URLSearchParams(window.location.hash.slice(1)).get('id'),
   );
+
   // This tree query is intentionally separate from the one in `MapLayout`,
   // since it kicks off only once on load!
-  const { data: initialLoadTreeData } = useTreeQuery(
-    { id: initialLoadTreeId.current },
-    { retry: 0, enabled: isMapLoaded && !!initialLoadTreeId },
+  useTreeQuery(
+    {
+      id: initialLoadTreeId.current,
+    },
+    {
+      retry: 0,
+      enabled: initialLoadTreeId.current != null,
```

Comment:
Ah crap, just realized there's a bug in here in that this query will be repeated after the default cache/stale time runs out. If that happens, the map will be recentered at seemingly random times. 

Maybe an alternative is having a hook for doing something just once after all conditions pass lol. Seems hacky but better than having a bunch of `isInitiated` flags laying around